system/lib/wasmfs/syscalls.cpp

```diff
@@ -1389,6 +1390,44 @@ int __syscall_fcntl64(int fd, int cmd, ...) {
   }
 }
 
+int doStatFS(std::shared_ptr<File>& file, size_t size, struct statfs* buf) {
```

Comment:
static?

Reply:
Sounds good, will fix other cases without it too.